slash/react/src/Accordion/CollapseCard.tsx

```diff
@@ -18,14 +21,27 @@ export const CollapseCard = ({
   id,
   open,
   onToggle,
+  className,
+  classModifier
 }: CollapseProps) => {
   const headerId = id;
 
+  let newClassModifier = open ? 'open' : '';
+  newClassModifier = classModifier
+    ? `${newClassModifier} ${classModifier}`
+    : `${newClassModifier}`;
+
+  const componentClassName = getComponentClassName(
+    className,
+    newClassModifier,
+    "af-accordion__details",
+  );
```

Comment:
Cela permet d'avoir un ternaire en moins, j'ai rajouté le trim dans le paramètre de getComponentClassName mais je pense que ce serait pas mal de l'avoir directement dans la fonction 🤔
```suggestion
  classModifier = ''
}: CollapseProps) => {
  const headerId = id;

  let newClassModifier = open ? 'open' : '';
  newClassModifier += ` ${classModifier}`;

  const componentClassName = getComponentClassName(
    className,
    newClassModifier.trim(),
    "af-accordion__details",
  );
```

Reply:
si tu ne passes pas de classModifier il te met une classe undefined 
![image](https://github.com/user-attachments/assets/f788a75c-cfb9-4c8d-9394-916a12c5e7fb)

Reply:
Oui, c'est pour ça que j'ai mis comme valeur par défaut à classModifier un empty string

Reply:
ah oui je n'avais pas vu cette modification là.

Reply:
corrigé